packit/source_git.py

```diff
@@ -395,14 +400,59 @@ def _put_downstream_sources(self):
         self.local_project.stage(self.dist_git.source_git_downstream_suffix)
         self.local_project.commit(message="add downstream distribution sources")
 
+    def _get_sources_url(self, tool):
+        try:
+            config = LookasideCacheHelper._read_config(tool)
+            base_url = config["lookaside"]
+        except (configparser.Error, KeyError) as e:
+            raise LookasideCacheError("Failed to read rpkg configuration") from e
+
+        package = self.dist_git.package_config.downstream_package_name
+        basepath = self.dist_git.local_project.working_dir
+
+        sources = list()
+        for source in LookasideCacheHelper._read_sources(basepath):
+
+            if tool == "fedpkg":
+                url = "{0}/{1}/{2}/{3}/{4}/{2}".format(
+                    base_url,
+                    package,
+                    source["filename"],
+                    source["hashtype"],
+                    source["hash"],
+                )
+            else:
```

Comment:
we'll need to make sure this works with CentOS Stream 9: https://gitlab.com/redhat/centos-stream/rpms/make/-/blob/c9s/sources

Reply:
AFAIK the implementation should work fine. It was inspired by `rebasehelper.helpers.lookaside_cache_helper`. It couldn't be used directly, because that implementation supports download of source files, but it does not support to get only urls of source files